### PR TITLE
Update riptide_vance_joy.txt

### PR DESCRIPTION
**Pull request by emuzychuk**

### CODE REVIEW

File: lyrics/riptide_vance_joy.txt [hunk 1]
```diff
@@ -1 +1,3 @@
+practice branch singer vance joy
+
```
Comment (Aanchal-Sagwal) [inline lyrics/riptide_vance_joy.txt:2]: example review

Comment (Aanchal-Sagwal) [inline lyrics/riptide_vance_joy.txt:2]:
```suggestion
yikes
```

**Review by Aanchal-Sagwal — APPROVED** [review]

**Comment by Aanchal-Sagwal** [comment]
whats the song again?

**Comment by emuzychuk** [comment]
> whats the song again?

Riptide!

**Pull request merged**